### PR TITLE
replace alert with snackbar

### LINKED ISSUE #1127: alert: error icon looks like close button
The error icon in alerts looks somewhat like a close icon:

Screenshot 2022-10-27 at 16 08 26

Suggest using something like ⚠️, ⛔️ or 😞.

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

### PR DESCRIPTION
**Pull request by wxtim**
These changes close #1127 
This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Appropriate tests are included (unit and/or functional).
- [x] No change log entry included - this is a like-for like functionality replacement of one vue component for another. Users do not need to made specifically aware of it.
- [x] No documentation update required.

### CODE REVIEW

**Comment by wxtim** [comment]
image
This is driving me crackers - it works locally.

File: tests/e2e/specs/layout.cy.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ describe('Default layout', () => {
     // visit any page first, so that we create the window.app reference
     cy.visit('/#/workflows/one')
     cy
-      .get('.v-alert')
+      .get('.v-snack__wrapper')
```
Comment (MetRonnie) [inline tests/e2e/specs/layout.cy.js:23]: One thing to try could be giving the snackbar component a unique `data-cy` attribute e.g. "alert-snack" and targeting `[data-cy=alert-snack] > .v-snack__wrapper'

**Comment by MetRonnie** [comment]
I'll have a look at some point

File: tests/e2e/specs/layout.cy.js [hunk 2]
```diff
@@ -20,8 +20,8 @@ describe('Default layout', () => {
     // visit any page first, so that we create the window.app reference
     cy.visit('/#/workflows/one')
     cy
-      .get('.v-alert')
-      .should('not.exist')
+      .get('.v-snack__wrapper')
+      .should('not.visible')
```
Comment (MetRonnie) [inline tests/e2e/specs/layout.cy.js:24]: Interesting that `be` (i.e. `not.be.visible`) is apparently optional

Comment (wxtim) [reply tests/e2e/specs/layout.cy.js:24]: Wasn't it you telling me it was syntactic sugar?

Comment (MetRonnie) [reply tests/e2e/specs/layout.cy.js:24]: I'm not too bothered

File: src/components/core/Alert.vue [hunk 3]
```diff
@@ -16,23 +16,27 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-      :type="alert.color"
-      :icon="alert.icon"
-      :class="getColor(alert.color)"
-      dismissible
-      tile
-      light
-      colored-border
-    >
-      <template v-slot:close="props">
-        <v-icon @click="closeAlert(props.toggle)">{{ svgPaths.close }}</v-icon>
```
Comment (datamel) [inline src/components/core/Alert.vue:31]: `closeAlert` is no longer called so this method can be deleted now I think?

`closeAlert` did clear the store, wonder if this needs replacing somewhere. I'll have a look into this now.

Comment (wxtim) [reply src/components/core/Alert.vue:31]: Did you come to any conclusions about this? I imagine I probably want to wire it back in?

File: src/components/core/Alert.vue [hunk 4]
```diff
@@ -52,7 +56,8 @@ export default {
       ]),
       svgPaths: {
```
Comment (datamel) [inline src/components/core/Alert.vue:57]: Think this svgPaths can go too now.

**Comment by wxtim** [comment]
@datamel  - I think it's safer to keep the original alert clearing, but you are right, the `toggleFunction` no longer has any purpose.

**Review by datamel — APPROVED** [review]
Manually tested. Working well, thanks @wxtim

**Comment by MetRonnie** [comment]
Thanks Tim, I've built upon this - superseded by #1218

**Pull request closed without merging**